pinaxcon/registrasion/management/commands/manualinvoice.py

```diff
@@ -7,21 +7,29 @@
 
 
 class Command(BaseCommand):
-    help = "Creates a manual invoice for multiple products against a user."
+    help = "Creates a manual invoice for multiple products against a user. Can specify multiple products."
 
     def add_arguments(self, parser):
         parser.add_argument('--user', required=True, help='user to add invoice to')
-        parser.add_argument('--product', required=True, help='product type of invoice')
-        parser.add_argument('--count', required=True, type=int, help='how many products')
+        parser.add_argument('--product', required=True, action="append", help='product name')
+        parser.add_argument('--count', required=True, action="append", type=int, help='how many products')
 
     def handle(self, *args, **options):
-        product = Product.objects.filter(name=options['product'])[0]
+        product_name = options['product']
+        count = options['count']
```

Comment:
COMPLETELY vulnerable to fuzz attack, here.   Yeah, this is a mgt command, only used by organizers / admins.   Still, would be better to first sanitize and then check to see that there actually IS such a product.